src/planner/World.cpp

```diff
@@ -151,5 +151,23 @@ std::mutex& World::getMutex() const
   return mMutex;
 }
 
+//==============================================================================
+bool World::equalConfiguration(const World* otherWorld) const
+{
+  if (otherWorld->mSkeletons.size() != mSkeletons.size())
+    return false;
+
+  for (size_t idx = 0; idx < mSkeletons.size(); ++idx)
+  {
+    if (mSkeletons[idx]->getName() != otherWorld->mSkeletons[idx]->getName())
```

Comment:
Should we be requiring that the `Skeleton`s are in the exact same order in the `World`s? Maybe we can allow permutations?

Reply:
The rationale for this was `World->getSkeleton(std::size_t i)` method. If `getSkeleton(i)` is used to get a skeleton and do something, it'd be better if the user can assume that `worlds` with `equalConfiguration` have the same ordering of skeletons.